Dockerfile

```diff
@@ -8,11 +8,13 @@ RUN gradle getMongoKafkaConnectDeps
 RUN curl -L "https://github.com/microsoft/ApplicationInsights-Java/releases/download/3.4.19/applicationinsights-agent-3.4.19.jar" \
     --output "applicationinsights-agent.jar"
 
-FROM quay.io/strimzi/kafka:0.38.0-kafka-3.6.0
-USER root:root
+FROM debezium/connect-base:2.4.1.Final@sha256:52297b4d15b2eff0a19b44337e34a9cd3144696a70393ab9f4eee14d697abb0e
 
-COPY --from=deps /deps/mongo-kafka-connect/ /opt/kafka/plugins/
+COPY --from=deps /deps/mongo-kafka-connect/ /kafka/connect/mongo-kafka-connect/
 COPY --from=deps /deps/applicationinsights-agent.jar .
 
 
-USER 1001
\ No newline at end of file
+USER root
+RUN chmod 777 -R /kafka/connect/
```

Comment:
Now that we know it works, we can optimize docker build by collapsing these two RUN into a single one, like `RUN chmod 777 -R /kafka/connect/ && chown kafka:kafka -R applicationinsights-agent.jar`